protos/peers.proto

```diff
@@ -162,6 +165,74 @@ message ElasticsearchConfig {
   optional string api_key = 5;
 }
 
+message IcebergConfig {
```

Comment:
consider iceberg.proto

Reply:
following the old pattern of having peers in peers.proto as UI, flow etc all use the same. JVM has its own proto called flow-jvm.